src/control-plane/backend/lambda/api/service/quicksight/reporting-check.ts

```diff
@@ -62,6 +63,10 @@ export class ReportingCheck {
           message: 'At least missing one of following parameters [dashboardId,sheetId,chartTitle,chartSubTitle].',
         };
       }
+      this.params.chartTitle = replaceSpecialCharacters(this.params.chartTitle);
```

Comment:
Let's keep the behavior the same as the frontend. Return a bad request if the input contains illegal characters.